eng/pipelines/common/variables.yml

```diff
@@ -39,7 +39,7 @@ variables:
 # We only run evaluate paths on runtime and runtime-community pipelines on PRs
 # keep in sync with /eng/pipelines/common/xplat-setup.yml
 - name: dependOnEvaluatePaths
-  value: ${{ and(eq(variables['Build.Reason'], 'PullRequest'), in(variables['Build.DefinitionName'], 'runtime', 'runtime-community', 'runtime-extra-platforms', 'runtime-wasm', 'runtime-wasm-libtests', 'runtime-wasm-non-libtests', 'runtime-ioslike', 'runtime-ioslikesimulator', 'runtime-android', 'runtime-androidemulator', 'runtime-maccatalyst', 'runtime-linuxbionic', 'dotnet-linker-tests', 'runtime-dev-innerloop', 'runtime-coreclr superpmi-replay', 'runtime-coreclr superpmi-diffs')) }}
```

Comment:
Is this change intentional? It doesn't seem related to what you're trying to achieve.

Reply:
This wasn't related to enabling hot/cold splitting, but just to skip these pipelines for now so they don't keep rerunning whenever I push a commit to this PR

Reply:
OK, I see, it's just a temporary instrumentation, we just need to make sure to remove it before merging in. I think a common practice is to mark PRs using such forms of temporary instrumentation as "WIP: xxx" (in the title) or as "NO-MERGE" (as a label), once we mark a PR ready for review we and / or the reviewers should make sure there are no remnants of such instrumentations left in the change. We hit this all the time including myself, it's kind of amazing how various logging outputs and other instrumentation changes can sneak into the final change that gets merged in, sometimes leading to surprises. Look at the amount of spew the ```src\tests\build``` command outputs for instance, I believe at least half of this is overlooked or unfixed msbuild / Roslyn spew.